datajunction-server/datajunction_server/database/column.py

```diff
@@ -39,6 +39,7 @@ class Column(Base):  # type: ignore
         insert_default=lambda context: labelize(context.current_parameters.get("name")),
     )
     type: Mapped[Optional[ColumnType]] = mapped_column(ColumnTypeDecorator)
+    description: Mapped[Optional[str]] = mapped_column()
```

Comment:
Should this be `mapped_column(String, default=None)` instead?

Reply:
I think nullability will be derived from the `Optional` type?

"In the absence of both of these parameters, the presence of typing.Optional[] within the [Mapped](https://docs.sqlalchemy.org/en/20/orm/internals.html#sqlalchemy.orm.Mapped) type annotation will be used to determine nullability, where typing.Optional[] means NULL, and the absence of typing.Optional[] means NOT NULL."

https://docs.sqlalchemy.org/en/20/orm/declarative_tables.html

Reply:
Ah true, this is not strictly required